find.go

```diff
@@ -347,6 +362,16 @@ outer:
 		return http.StatusNotFound, nil
 	} else {
 		latencyTags = append(latencyTags, tag.Insert(metrics.Found, "yes"))
+		if foundCaskade {
+			latencyTags = append(latencyTags, tag.Insert(metrics.FoundCaskade, "yes"))
+		} else {
+			latencyTags = append(latencyTags, tag.Insert(metrics.FoundCaskade, "no"))
+		}
+		if foundRegular {
+			latencyTags = append(latencyTags, tag.Insert(metrics.FoundRegular, "yes"))
+		} else {
+			latencyTags = append(latencyTags, tag.Insert(metrics.FoundRegular, "no"))
+		}
```

Comment:
```suggestion
		yesno := func(yn bool) string {
			if yn {
				return "yes"
			}
			return "no"
		}	
		latencyTags = append(latencyTags, tag.Insert(metrics.FoundCaskade, yesno(foundCaskade)))
		latencyTags = append(latencyTags, tag.Insert(metrics.FoundRegular, yesno(foundRegular)))
```

Reply:
That looks better :) One thing I miss from Java - is ternary operators. I.e. `foundCaskade ? "yes" : "no"`